transfer.md

```diff
@@ -1,16 +1,23 @@
 # CRI-O Usage Transfer
 
-This document outlines useful information for ops and dev transfer as it relates to infrastructure that utilizes CRI-O.
+This document outlines useful information for operations and development transfer as it relates to infrastructure that utilizes CRI-O.
 
 ## Operational Transfer
 
 ## Abstract
 
-The `crio` daemon is intended to provide the [CRI](https://github.com/kubernetes/community/blob/master/contributors/devel/container-runtime-interface.md) socket needed for Kubernetes to use for automating deployment, scaling, and management of containerized applications (See the document for [configuring kubernetes to use CRI-O](./kubernetes.md) for more information on that).
-Therefore the [crictl](https://github.com/kubernetes-incubator/cri-tools) command line is a client that interfaces to the same grpc socket as the kubernetes daemon would, for talking to the `crio` daemon.
-In many ways [crictl](https://github.com/kubernetes-incubator/cri-tools) is only as feature rich as the Kubernetes CRI requires.
+The `crio` daemon is intended to provide the [CRI][0] socket needed for Kubernetes to use for automating deployment, scaling, and management of containerized applications (See the document for [configuring kubernetes to use CRI-O](./kubernetes.md) for more information on that).
```

Comment:
minor nit of  a nit.  suggest dropping 'on that' at the end.

Reply:
Ditto the 'configuring kubernetest to use CRI-O' link

Reply:
Fixed both :+1: